cmd/manager/main.go

```diff
@@ -129,7 +129,6 @@ func main() {
 	// Configure metrics. If it errors, log the error but continue.
 	if err := metrics.ConfigureMetrics(context.TODO(), *metricsServer); err != nil {
 		log.Error(err, "Failed to configure Metrics")
-		os.Exit(1)
```

Comment:
I removed this so the operator can be run in minikube. It means that metrics failures will be non-fatal, which complies with the team's standards for metrics. Also makes behavior match the comment above it.

Reply:
I thought we had a discussion about this as a team, and while I agreed with you that it's better to have metrics failures be fatal, we were supposed to make them non-fatal? I'll update the operator guidelines to clarify this, and put the `os.Exit(1)` back.

Reply:
We had a more recent discussion where it came up again and the consensus changed to making them fatal. I put this line in PRs across all the hive operators and they now all have it. Thanks for updating the guidelines doc, I forget that it exists sometimes.